src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -394,6 +394,8 @@
   "IndexerSettingsPasskey": "Pass Key",
   "IndexerSettingsQueryLimit": "Query Limit",
   "IndexerSettingsQueryLimitHelpText": "The number of max queries as specified by the respective unit that {appName} will allow to the site",
+  "IndexerSettingsRejectBlocklistedTorrentHashes": "Reject Blocklisted Torrent Hashes While Grabbing",
+  "IndexerSettingsRejectBlocklistedTorrentHashesHelpText": "If a torrent is blocked by hash it may not properly be rejected during RSS/Search for some indexers, enabling this will allow it to be rejected after the torrent is grabbed, but before it is sent to the client.",
```

Comment:
@mynameisbogdan , @bakerboy448  - thank you both for your help on this (and your work on prowlarr generally)!
I know it's off topic, but maybe you find this interesting for your own media servers, which keeps the torrent queue free of stuck items: 
https://github.com/ManiMatter/decluttarr

The setting we are adding here (and obv. the underlying settings in radarr/sonarr etc) directly benefit this app. 
Just wanted to share, in case you can benefit from it

Reply:
wording  lgtm